scripts/parcel-dev-reporter.js

```diff
@@ -0,0 +1,64 @@
+const { Reporter } = require('@parcel/plugin');
+const { fork } = require('child_process');
```

Comment:
This is a small plugin to relaunch the server whenever you update the code. It's much more consistent than something like nodemon and not too hard to read.